website/src/App.jsx

```diff
@@ -46,6 +46,9 @@ class App extends PureComponent {
             <Route path="/signin" component={Signin} />
             <Route path="/create/class" component={CreateClass} />
             <Route path="/join/class" component={JoinClass} />
+            <Dashboard>
```

Comment:
We no longer use this method wait until the ProtectedView pr is merged and follow that structure